src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -250,6 +250,21 @@ public async Task InspectLocalsInPreviousFramesDuringSteppingIn(bool use_cfo)
             );
         }
 
+        [Fact]
+        public async Task InspectNullLocalsDuringSteppingIn()
```

Comment:
Which null local is being tested here?

Reply:
I renamed it and added another explicit test for the other null